mujoco_ros/include/mujoco_ros/lodepng.h

```diff
@@ -118,8 +122,7 @@ or comment out LODEPNG_COMPILE_CPP below*/
 
 #ifdef LODEPNG_COMPILE_PNG
 /*The PNG color types (also used for raw image).*/
-typedef enum LodePNGColorType
-{
+using LodePNGColorType = enum LodePNGColorType {
```

Comment:
I think you can omit the using statement in all these type definitions as the type is already directly accessible with the given name.

Reply:
I tried removing the using statements but that causes C errors during compilation. I'll leave it as is for now